dns/README.md

```diff
@@ -29,6 +29,66 @@ We host it in Google Cloud DNS.
   * GCP project = kubernetes-public
   * https://console.cloud.google.com/net-services/dns/zones?project=kubernetes-public&organizationId=758905017065
 
+## Requesting a DNS update
+
+The process for requesting an update uses Github Issues and PRs.
+
+### Update Request Issue
+
+Open a new issue on this repository with the title "DNS Update Request"
+
+In the issue, please list the following details:
+   * If this update is a create, delete or update.
+   * The base domain that is being modified (e.g. "k8s.io")
+   * The complete data for the existing DNS record if applicable (Updates, Deletes).
+   * The complete data for the new DNS record if applicable (Creates, Updates).
+   * The reason for the update.
+
+Once this issue is created, it should be acknowledged by a DNS administrator.
+
+### Example update issue content:
```

Comment:
I think this work well as an issue template. Move this content to .github/ISSUE_TEMPLATE/dns-request.md

Start that file with something like:
```
name: DNS Update Request
about: Request a create, delete or update of a kubernetes.io or k8s.io DNS record
 ---

(then all of your example content / instructions here)
```

This has worked pretty well for us at k/org, eg: https://github.com/kubernetes/org/issues/new?template=membership.md

I think you would also need to add a default / "general" issue template, we went with https://raw.githubusercontent.com/kubernetes/org/master/.github/ISSUE_TEMPLATE/general-support.md but honestly I think you wouldn't need the add'l content just the name:/about:

Reply:
Added.